package.json

```diff
@@ -9,6 +9,9 @@
 		"node": "^14 || ^16 || >=18"
 	},
 	"workspaces": [
+		"packages/css-tokenizer",
+		"packages/css-parser-algorithms",
+		"packages/media-query-list-parser",
 		"packages/*",
```

Comment:
Given L15, do we need to specify in particular? I notice we have progressive too so I wasn't sure.

Reply:
This is a "hack" to specify the build and test order.
npm workspaces do not build a dependency graph between local packages.